server/deployster_service.go

```diff
@@ -37,13 +38,16 @@ func NewDeploysterService(listen string, version string, username string, passwo
 
 func (ds *DeploysterService) ConfigureRoutes() {
 	fleetClient := fleet.NewClient("/var/run/fleet.sock")
+	dockerClient, _ := docker.NewClient("unix:///var/run/docker.sock")
```

Comment:
We need to start moving these types of things into configuration's of some sort.